packages/clay-color-picker/src/Splotch.tsx

```diff
@@ -33,17 +33,16 @@ const ClayColorPickerSplotch = React.forwardRef<HTMLButtonElement, IProps>(
 		return (
 			<button
 				{...otherProps}
-				className={`btn clay-color-btn ${className ? className : ''}`}
+				className={`${active ? 'active ' : ''}btn clay-color-btn ${
+					requireBorder ? 'clay-color-btn-bordered ' : ''
+				}${className ? className : ''}`}
 				ref={ref}
 				style={{
-					...(active ? {outline: 'auto 3px #55ADFF'} : {}),
-					...(requireBorder
-						? {border: '1px solid #E7E7ED'}
-						: {borderWidth: 0}),
 					background: `#${value}`,
 					height: size,
 					width: size,
 				}}
+				tabIndex={active ? -1 : undefined}
```

Comment:
Thinking better, maybe we don't even need it anymore.

Reply:
Where can I find documentation on FocusScope? I can only find an issue on reactjs/rfcs. I'm just trying to take the active button out of the tab order. We could always use `span`/`div` but would be nice if there was a way to take a button out of tab order.

Maybe we could create another issue for managing focus, there are some other things I would like to make better.

Reply:
@pat270 `<FocusScope />` is our own implementation to deal with the Focus of the elements when they are using `React.Portal`, unfortunately, we don't have documentation of it because we created it for internal use. https://github.com/liferay/clay/blob/master/packages/clay-shared/src/FocusScope.tsx

> I'm just trying to take the active button out of the tab order. We could always use span/div but would be nice if there was a way to take a button out of tab order.

Hmm, I think I understand now, do you want to ignore the active element in the order of the tabs, in the tab should you skip the active element to the next one?

In this case, FocusManager will ignore non-focusable and `disabled` elements.

> Maybe we could create another issue for managing focus, there are some other things I would like to make better.

It might be better to create an issue, to describe how we can handle the focus in ColorPicker.

Reply:
Sounds good I'll make an issue.